examples/handling/georeferencing/crop_raster.py

```diff
@@ -0,0 +1,51 @@
+"""
+Crop a raster
+=============
+
+This example demonstrates the cropping of a raster using :func:`geoutils.Raster.crop`.
+"""
+# sphinx_gallery_thumbnail_number = 2
```

Comment:
This does not render properly.

Reply:
Solved in [9d487d0](https://github.com/GlacioHack/geoutils/pull/351/commits/9d487d0a85fe2acffbeb4b75d2e3765d23090f66)